### DISCUSSION TITLE
New Sonoff T0 EU 1 gang ghost switching free?

### DISCUSSION THREAD

**Discussion by henkiejan1** [arendst/Tasmota#13115] [Q&A] [answered]
Hi everyone,

Every switch in my home is a Sonoff. A time ago did a switch die so i bought a new one. But this one has sadly ghost switching. By changing some options it works better now but makes me not happy. Now i want to buy a new one for bedroom and i don't like to wake a few times in my night to switch of the light. ;) 

So my question is: did someone bought a sonoff T0 EU 1 gang recently from the official Sonoff store on Aliexpress and did you just have ghost switching? I hope to hear from you :)

Thank you very much!

**Comment by henkiejan1** [comment]
No one any experience recently??

**Reply by TrDA-hab** [reply]
Sonoff D1 has a similar problem!

If you’re having “ghost switching” events:
1. Pairing a RF remote with  Sonoff T0 could stop them.
2. Or remove RF 433MHz chip.

**Reply by henkiejan1** [reply]
The Sonoff T0 doesn't have a RF433 module. So that could sadly the solution. But maybe i need to give a new one a try.

**Comment by barbudor** [comment, marked as answer]
@henkiejan1, Set `Weblog 3` and report the logs when ghost switching occurs
